pkg/git/repository.go

```diff
@@ -136,16 +140,30 @@ func (o *CloneOptions) Clone(ctx context.Context) (Repository, fs.FS, error) {
 
 	r, err := clone(ctx, o)
 	if err != nil {
-		if err == transport.ErrEmptyRemoteRepository {
-			log.G(ctx).Debug("empty repository, initializing new one with specified remote")
+		switch err {
+		case transport.ErrRepositoryNotFound:
+			if !o.AutoCreate {
+				return nil, nil, err
+			}
+
+			log.G(ctx).Debug("no repository, creating a new one")
```

Comment:
let's make some non debug message, this information is dramatic enough to make it `info` level. You can change the massage to: `repository '%s' was not found, trying to create it...`